src/app/settings/components/settings/settings.component.scss

```diff
@@ -4,6 +4,7 @@
   display: flex;
   &__content {
     flex-grow: 1;
+    background-color: var(--gray-color);
```

Comment:
This background color is too dark. Look at the mockup to see how it should look like